File: app/src/main/java/org/mesonet/app/maps/MapGroupRecyclerViewAdapter.java
package org.mesonet.app.maps;

import org.mesonet.app.baseclasses.BaseActivity;



public class MapGroupRecyclerViewAdapter extends MapsRecyclerViewAdapter {
    public MapGroupRecyclerViewAdapter(BaseActivity inBaseActivity) {
        super(inBaseActivity);
    }


    @Override
    public int getItemCount()
    {
        if(mDataItems != null && mDataItems.size() > 4)
            return 3;

        return super.getItemCount();
    }
}

File: app/src/main/java/org/mesonet/app/maps/MapGroupRecyclerViewAdapter.kt
package org.mesonet.app.maps

import org.mesonet.app.baseclasses.BaseActivity


class MapGroupRecyclerViewAdapter(inBaseActivity: BaseActivity) : MapsRecyclerViewAdapter(inBaseActivity) {
    override fun getItemCount() : Int
    {
        return if (mDataItems != null && mDataItems!!.size > 4) 3 else super.getItemCount()
    }
}
